dev-support/hbase-personality.sh

```diff
@@ -567,6 +567,13 @@ function hadoopcheck_rebuild
     else
       hbase_hadoop2_versions="2.7.1 2.7.2 2.7.3 2.7.4 2.7.5 2.7.6 2.7.7"
     fi
+  elif [[ "${PATCH_BRANCH}" = branch-1 ]]; then
+    yetus_info "Setting Hadoop 2 versions to test based on branch-1 rules."
+    if [[ "${QUICK_HADOOPCHECK}" == "true" ]]; then
```

Comment:
Is the if-else necessary as `hbase_hadoop2_version` will be set to the same versions, regardless of the check? Or is it just to stay consistent?

Reply:
Intenional, just want to keep consistent...